utils/hostapi.d.ts

```diff
@@ -128,6 +129,7 @@ export interface PickAppResourceOptions extends IAzureQuickPickOptions {
 export interface AppResource {
     readonly id: string;
     readonly name: string;
+    readonly azureResourceType: AzureResourceType;

```

Comment:
We should add a doc comment explaining that this property is a custom derived one that we added to the data.